src/BlocklyInterface/robotblocks/mechanisms/MechanismInput.ts

```diff
@@ -0,0 +1,33 @@
+import { addCustomBlock } from "../AddBlockUtil";
+
+export function addMechanismInputBlock() {
```

Comment:
The blocks should more closely map to the set and get digital commands. The fact that the channel is actually controlled by the grabber mechanism is abstracted from the programmer in this case